pages/advanced/content/ssr.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   title: 'Server Side Rendering',
   subtitle: '',
-  description: `testing`.replace(/~/g, '`'),
+  description: `Because both ~glamor~ and ~react~ support SSR, glamorous does too! I actually do this on [my personal site](https://github.com/kentcdodds/kentcdodds.com) which is generated at build-time on the server. Learn about rendering [~react~ on the server](react on the server) and [~glamor~ too](https://github.com/threepointone/glamor/blob/5e7d988211330b8e2fca5bb8da78e35051444efd/docs/server.md).
```

Comment:
Could you put this on multiple lines? Break it every 80 characters or so... Thanks!